tests/ci/run_fips_callback_tests.sh

```diff
@@ -70,19 +70,31 @@ function run_all_break_tests() {
 }
 
 echo "Testing AWS-LC static breakable build with custom callback and Jitter enabled"
-build_and_test -DFIPS=1 \
-  -DCMAKE_C_FLAGS="-DBORINGSSL_FIPS_BREAK_TESTS -DAWSLC_FIPS_FAILURE_CALLBACK" \
+build_and_test -DCMAKE_BUILD_TYPE=Release \
+  -DCMAKE_BUILD_SHARED_LIBS=OFF \
+  -DFIPS=1 \
```

Comment:
NP: This test can only be run on Linux (i.e., targets for which we have a FIPS static build). 

Do we need to test targets for which we produce a FIPS shared build?

Reply:
> Do we need to test targets for which we produce a FIPS shared build?

ACCP always links AWS-LC statically, so not as far as ACCP is concerned.